src/app/component/general/eco-news/filter-news/filter-news.component.ts

```diff
@@ -0,0 +1,61 @@
+import { Component, OnInit, Output, EventEmitter } from '@angular/core';
+
+@Component({
+  selector: 'app-filter-news',
+  templateUrl: './filter-news.component.html',
+  styleUrls: ['./filter-news.component.css']
+})
+
+export class FilterNewsComponent implements OnInit {
+
+  private filters = [{ name: 'news', isActive: false },
+  { name: 'events', isActive: false },
+  { name: 'courses', isActive: false },
+  { name: 'initiatives', isActive: false },
+  { name: 'ads', isActive: false }];
+
+  @Output() gridOutput = new EventEmitter<Array<string>>();
+
+  constructor() { }
+
+  ngOnInit() {
+    this.emitter();
+  }
+
+  private emitter(): void {
```

Comment:
Please rename method name, 'cause it's not clear

Reply:
Fixed